style/style.css

```diff
@@ -533,10 +527,628 @@
+.button-order-mobile,
+.mobile-movee-order,
+.btn-mobile-order,
+.footer-mobile-order {
+    display: none;
+}
+
+.order-box {
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    align-items: center;
+}
+
+.order-box h2 {
+    font-weight: 600;
+    font-size: 1.875rem;
+    line-height: 2.3125rem;
+    margin-top: 7.75rem;
+}
+
+.additional-services-text {
+    margin-top: 3.75rem;
+    font-weight: bold;
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+    padding-right: 28.125rem;
+}
+
+.call-back-text {
+    font-size: 1rem;
+    line-height: 1.3125rem;
+    margin-top: 0.625rem;
+}
+
+.order-box {
+    margin: 0 15%;
+    position: relative;
+}
+
+.make-order_info {
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+    align-items: center;
+    margin-bottom: 4rem;
+    flex-wrap: wrap;
+}
+
+.make-order-location {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.make-order-location-box {
+    display: flex;
+    flex-direction: column;
+    margin-left: 2.1875rem;
+}
+
+.make-order-location-box:first-child {
+    margin-right: 1.875rem;
+}
+
+.user-data {
+    line-height: 2.6rem;
+    border-bottom: 0.0625rem solid #333333;
+    width: 18.75rem;
+    margin-bottom: 0.3rem;
+    display: flex;
+    flex-direction: row;
+    margin-top: 5.0625rem;
+}
+
+.tel {
+    margin-left: 6.875rem;
+}
+
+.city-add {
+    line-height: 2.6rem;
+    border-bottom: 0.0625rem solid #333333;
+    max-width: 18.75rem;
+    margin-bottom: 0.3rem;
+    margin-top: 1.875rem;
+}
+
+.city-set {
+    font-weight: bold;
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+    margin-bottom: 4.3125rem;
+}
+
+.city-select {
+    width: 18.75rem;
+    border-bottom: 1px solid #333333;
+    margin-bottom: 1.25rem;
+    margin-right: 3.125rem;
+}
+
+.payment-method {
+    margin-top: 4.25rem;
+    font-weight: bold;
+    font-size: 1.25rem;
+    line-height: 1.5rem
+}
+
+.payment-method-box {
+    margin-top: 2.125rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-left: 3.75rem;
+}
+
+.choice-of-payment-method {
+    margin-right: 2.5rem;
+    font-size: 1rem;
+    line-height: 1.3125rem;
+}
+
+.additional-services-text {
+    margin-top: 3.75rem;
+    font-weight: bold;
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+    padding-right: 32%;
+}
+
+.additional-services-text::after {
+    content: url(../img/Vector\ 2.png);
+    margin-left: 0.625rem;
+}
+
+.input-radio-payment {
+    margin-right: 3.125rem;
+}
+
+.additional-services-item {
+    display: flex;
+    flex-direction: row;
+    margin-left: 3.125rem;
+    margin-top: 3.125rem;
+    position: relative;
+}
+
+.additional-services-wrapper {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    max-width: 56.25rem;
+    margin-left: 5.0625rem;
+}
+
+.payment-method-box {
+    margin-right: 2.35rem;
+}
+
+
+/* radio-method-box*/
```

Comment:
We don't use comments in the code. I already talked about this, why aren't you listening to me?